### PR TITLE
new example based on Grabowski & Pawlowska 2023 (GRL)

### PR DESCRIPTION
**Pull request by AgnieszkaMakulska**

### CODE REVIEW

**Comment by slayoo** [comment]
closing and reopening to trigger build against current main branch

**Comment by AgnieszkaMakulska** [comment]
- [ ] smoke test for figures 1,2,3
- [ ] unit test for concentration
- [ ] check out ripening product

**Comment by slayoo** [comment]
- [ ] tests for the order of lines on the plot (e.g., blue is below red, etc)

File: PySDM/products/impl/concentration_product.py [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
 class ConcentrationProduct(MomentProduct):
     def __init__(self, *, unit: str, name: str, specific: bool, stp: bool):
+        """
+        SDM toggles expressing the concentration in terms of standard temperature
```
Comment (slayoo) [inline PySDM/products/impl/concentration_product.py:12]: SDM -> stp

File: PySDM/products/size_spectral/mean_radius_activated.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"""
+mean radius of particles within a grid cell, for activated, unactivated of both
```
Comment (slayoo) [inline PySDM/products/size_spectral/mean_radius_activated.py:2]: "of both" -> "or both"

File: PySDM/products/size_spectral/mean_volume_radius.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+"""
+mean volume radius of particles within a grid cell, for activated, unactivated of both
```
Comment (slayoo) [inline PySDM/products/size_spectral/mean_volume_radius.py:2]: "of both" -> "or both"

File: PySDM/products/size_spectral/particle_concentration_activated.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
+"""
+concentration of particles within a grid cell (either per-volume of per-mass-of-dry air),
+ activated, unactivated of both
```
Comment (slayoo) [inline PySDM/products/size_spectral/particle_concentration_activated.py:3]: ditto

File: examples/PySDM_examples/Grabowski_and_Pawlowska_2023/simulation.py [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+from PySDM.physics import si
+
+
+class Simulation(BasicSimulation):
+    def __init__(
+        self,
+        settings,
+        products=None,
+        scipy_solver=False,
+        rtol_thd=1e-10,
```
Comment (slayoo) [inline examples/PySDM_examples/Grabowski_and_Pawlowska_2023/simulation.py:20]: worth checking if condensation defaults are not enough? (or even lower!)

**Review by slayoo — APPROVED** [review]

**Pull request merged**